web-services/common-util/src/main/java/datawave/security/util/AuthorizationsUtil.java

```diff
@@ -83,11 +108,17 @@ public static Set<Authorizations> getDowngradedAuthorizations(String requestedAu
      *            thrown.
      * @param principal
      *            The principal from which to retrieve entity authorizations.
+     * @param userService
+     *            The remote user service if any which will be used to determine the remote authorizations. If this is null, then the auths in the supplied
+     *            principal will be used. Otherwise this service will be used to retrieve the remote principal from which the auths will be determined.
      * @return A set of {@link Authorizations}, one per entity represented in {@code principal}. The user's auths are replaced by {@code requestedAuths} so long
      *         as the user actually had all of the auths. If {@code requestedAuths} is {@code null}, then the user's auths are returned as-is.
      */
-    public static LinkedHashSet<Authorizations> getDowngradedAuthorizations(String requestedAuths, DatawavePrincipal principal) {
-        
+    public static LinkedHashSet<Authorizations> getDowngradedAuthorizations(String requestedAuths, DatawavePrincipal principal, RemoteUserOperations userService)
+                    throws AuthorizationException {
+        if (userService != null) {
+            principal = userService.getRemoteUser((DatawavePrincipal) principal);
```

Comment:
```suggestion
            principal = userService.getRemoteUser(principal);
```
principal already is a DatawavePrincipal